src/marqo/s2_inference/model_registry.py

```diff
@@ -276,7 +276,8 @@ def _get_open_clip_properties() -> Dict:
              'dimensions': 1024,
              'note': 'open_clip models',
              'type': 'open_clip',
-             'pretrained': 'laion2b_s12b_b42k'},
+             'pretrained': 'laion2b_s12b_b42k',
+             'model_size': 5,},
```

Comment:
Interesting. Was this model a lot larger than its peers? What method should we use the figure this out? Manual testing for the most commonly used models?